modules/local/wipertools/wipe.nf

```diff
@@ -0,0 +1,46 @@
+process WIPER {
+    tag "$meta.id"
+    label 'process_single'
+    container 'docker.io/mazzalab/fastqrepair_nf_env:1.0.1'
+
+    input:
+        tuple val(meta), path(fastq)
+
+    output:
+        tuple val(meta), path("*_wiped.fastq.gz"), emit: fixed_fastq
+        tuple val(meta), path("*_report.txt")    , emit: report
+        path "versions.yml"                      , emit: versions
+
+    when:
+        task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def filename = "${fastq.baseName}"
+    def VERSION = '1.0.1' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
```

Comment:
Just a tohought: if you do opt  for putting the package on bioconda, might be good to make a 1.0.2 which includes a `--version` parameter :) (But I recognise we are asking more and more, from you now  so thisi s just a suggestion!)